src/Twilio.Api/TransportException.cs

```diff
@@ -5,11 +5,11 @@
 
 namespace Twilio
 {
-    public class TransportException<T> : Exception

```

Comment:
The only place we are catching the TransprotException is here. We dont print out the generic type in any of the logging. We should be safe to remove this. https://github.com/ZocDoc/zocdoc_web/blob/069803ce9f787a4594628a722542d5917fb72b9f/Messaging/ZocDoc.Messaging.Sending.Impl/Sms/TwilioRestClientExceptionThrowingDecorator.cs#L71